flow_py_sdk/tx.py

```diff
@@ -70,6 +70,14 @@ def __init__(
         self.key_sequence_number: int = key_sequence_number
 
 
+class SignatureTemp(object):
```

Comment:
This does not need to be exposed, so rename to `_SignatureTemp` but it also should probably be called something like `_TxSigner`?